web_monitoring/diffing_server.py

```diff
@@ -12,6 +14,9 @@
 import web_monitoring.html_diff_render
 import web_monitoring.links_diff
 
+if os.environ.get('LOG_LEVEL'):
+    level = os.environ['LOG_LEVEL'].upper()
+    logging.basicConfig(level=getattr(logging, level))
```

Comment:
DID NOT KNOW SO NICE!

Reply:
Side note: I debated using `APP_ENV` (which I think we’d talked about using to set the [Tornado server’s debug mode](http://www.tornadoweb.org/en/stable/guide/running.html#debug-mode) or some other less-specific variable name. Any thoughts on that, or does `LOG_LEVEL` feel good?

Should I also add this to `cli.py`? Is this and that the right place to put it, or do you think it belongs in the `scripts/*` files rather than the modules here?

Reply:
I think `LOG_LEVEL` is good: easy to understand what is does and precisely targeted. If we want another env that makes coordinated changes to the module's behavior, we can always check for multiple env vars here.

I usually do `basicConfig` in the `__init__.py` of the package, and do `logging.getLogger(__name__)` in each module.

Reply:
Right, exactly.